apps/backend/app/api/routes/transformations.py

```
@@ -1511,4 +1522,372 @@ async def clear_history(
+        )
+
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.exception(f"Error listing bulk jobs: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error") from e
+
+
+@router.post("/datasets/{dataset_id}/bulk-jobs/{job_id}/cancel")
+async def cancel_bulk_job(
+    dataset_id: str,
```

Ruff (0.14.10)

Unused function argument: `dataset_id`

(ARG001)